.eslintrc.js

```diff
@@ -6,10 +6,22 @@ module.exports = {
     "plugin:markdown/recommended",
   ],
   overrides: [
+    {
```

Comment:
Can we revert the changes to the config for now? We're going to review them internally as we all have some opinions with the new rules as they are.